### PR TITLE
python@3.13: modify sysconfig

### PR DESCRIPTION
**Pull request by lukeshingles**
Based on https://github.com/Homebrew/formula-patches/commit/6d2fba8de3159182025237d373a6f4f78b8bd203 with minor changes for updated source code in python 3.13.

Needed for https://github.com/Homebrew/homebrew-core/pull/182840

### CODE REVIEW

File: python/3.13-sysconfig.diff [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     scheme = _get_preferred_schemes()[key]
+     if scheme not in _INSTALL_SCHEMES:
+@@ -348,6 +358,7 @@ def _init_non_posix(vars):
+     """Initialize the module as appropriate for NT"""
+     # set basic install directories
+     import _winapi
++
+     import _sysconfig
+     vars['LIBDEST'] = get_path('stdlib')
+     vars['BINLIBDEST'] = get_path('platstdlib')
```
Comment (Bo98) [inline python/3.13-sysconfig.diff:53]:
```suggestion
```

**Review by Bo98 — APPROVED** [review]

File: python/3.13-sysconfig.diff (hunk 1 above)
Comment (lukeshingles) [reply python/3.13-sysconfig.diff:53]: I don't understand what  this doing. Should I test this change on CI with an inline patch for https://github.com/Homebrew/homebrew-core/pull/182840 or just commit it?

**Comment by lukeshingles** [comment]
Ok, suggestion incorporated and ready!

**Comment by Bo98** [comment]
Thanks!

**Pull request merged**